README.md

```diff
@@ -429,4 +425,178 @@ No token or bad token response
 }
 ```
 
+### Contacts
+
+Get login credentials: <br>
+`Refer to Companies "Get login credentials" above`
+
+#### Get all contacts for a user
+Request:
+
+```
+GET /api/v1/users/:user_id/contacts
+
+Authorization: Bearer Token - put in token for user
+```
+Successful Response:
+
+```
+{
+  "data": [
+    {
+      "id": "1",
+      "type": "contacts",
+      "attributes": {
+        "first_name": "John",
+        "last_name": "Smith",
+        "company": "Turing",
+        "email": "123@example.com",
+        "phone_number": "(123) 555-6789",
+        "notes": "Type notes here...",
+        "user_id": 4
+      }
+  },
+  {
+    "id": "2",
+    "type": "contacts",
+    "attributes": {
+      "first_name": "Jane",
+      "last_name": "Smith",
+      "company": "Turing",
+      "email": "123@example.com",
+      "phone_number": "(123) 555-6789",
+      "notes": "Type notes here...",
+      "user_id": 4
+      }
+    }
+  ]
+}
+```
+Successful response for users without saved contacts:
+
+```
+{
+  "data": [],
+  "message": "No contacts found"
+}
+```
+
+#### Create a contact with required and optional fields.
+New contacts require a unique first and last name. All other fields are optional.
+
+Request:
+```
+POST /api/v1/users/:user_id/contacts
```

Comment:
New nested path structure and all subsequent tests look solid and in line with today's discussion.